docs/how-to/understanding-break-the-monolith.md

```diff
@@ -27,33 +27,37 @@ If an entity in your original monolith service has a non-relation field like `cr
 
 ### Handling One-to-One Relations
 
-In one-to-one relationships, both entities involved in the relationship retain a single field representing the remote entity's ID. This approach maintains a link between the entities even after they are distributed across different services.
+In one-to-one relationships, both entities involved in the relationship undergo a modification where their related field names are suffixed with `Id`. This approach maintains a virtual link between the entities. This means that while the entities reside in separate databases, the suffixed `Id` fields provide a means to reference each other virtually rather than through a direct database relationship.
```

Comment:
I consulted with Muly and he said that virtual is not the best term here (- sorry for the confusion).
He suggested to use "an external key" instead.